app/__init__.py

```diff
@@ -47,10 +35,9 @@ def clean_tags_and_replace_highlight_tag(text, highlight_tag):
 
 def format_opensearch_field_name(field):
     """Format the name of an OpenSearch field using a map or dynamically"""
-    if field in opensearch_field_name_map:
-        return opensearch_field_name_map[field]
-    else:
-        return field.replace("_", " ").capitalize()
+    return opensearch_field_name_map.get(
+        field, field.replace("_", " ").capitalize()
```

Comment:
why dont we remove this and just use the reverse lookup of the dict we have to be certain of how we are presenting in the UI?